widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.kt

```diff
@@ -359,6 +361,12 @@ internal class ChatController(
         mediaUpgradeOfferRepositoryCallback?.let { removeMediaUpgradeCallbackUseCase(it) }
     }
 
+    fun onImageItemClick(item: AttachmentFile) {
+        if (isFileReadyForPreviewUseCase(item)) {
```

Comment:
I'd suggest showing some `Toast` in a case when the file is not yet ready, to provide some feedback to the user.

Reply:
fixed 45537a8b2ebe0acb61caa153d8e805ed7723169b

Reply:
Ideally, I think we need to show some placeholder until preview is available. I think so because the space is currently empty until preview is loaded. From a visitor point of view, it looks like the image hasn't been sent or disappeared.
If you agree with this point, I can raise this question in Slack and create a separate ticket for later (if we agree to make changes).

Reply:
I agree, I even would suggest having a `shimmer effect` as a placeholder, which will indicate that image loading is in progress. But I think it should be implemented for both platforms.

![1_Tu5jffgXlt_aP-uVPKqKng](https://github.com/salemove/android-sdk-widgets/assets/19994238/0c867d53-fb44-4014-a061-0d4f36b1df04)

Reply:
Agree with you. I will rise the topic in slack.